### ISSUE TITLE
CSS file incorrectly warning about missing html lang declaration.

### ISSUE THREAD

**Issue by NecroBones** [validator/validator#628]
I've been sending a CSS file to the validator, and it's throwing an error about HTML doctypes missing a language, even though it's CSS. It claims that line 0 is missing an html language declaration, even though as CSS, there's no DOCTYPE or title as shown (see below).

```
WARNING: Consider adding a lang attribute to the html start tag to
declare the language of this document. 

From line 0, column 16; to line 0, column 22 

<!DOCTYPE html><TITLE>s</tit
```

**Comment by sideshowbarker** [comment]
Thanks for taking time to report this — it’s a bug that I see now I accidentally introduced in an unrelated change I made just a couple days ago, and I’ll get it fixed as soon as possible (by tomorrow if not today)

**Comment by sideshowbarker** [comment]
I’ve pushed the fix to https://validator.w3.org/nu/

Thanks for using the CSS-document-checking feature — until you reported this bug, I wasn’t sure anybody else has been using it yet except me…

**Comment by NecroBones** [comment]
My pleasure, glad to help.
